Gemfile

```diff
@@ -23,7 +23,7 @@ gem 'dotiw', '>= 4.0.1'
 gem 'faraday'
 gem 'faraday_middleware'
 gem 'foundation_emails'
-gem 'good_job', '~> 2.7.0'
+gem 'good_job', '~> 2.99.0'
```

Comment:
❤️ thank you! I was just thinking it might be time to update

Reply:
👍🏼 

We have a major version update following this: https://github.com/bensheldon/good_job#upgrading-v2-to-v3